packages/web/styles/globals.css

```diff
@@ -22,6 +22,9 @@
   ::-webkit-scrollbar-corner {
     background: rgba(0, 0, 0, 0);
   }
+  .tokendetailshadow::after {
```

Comment:
Rationale for putting this in globals.css instead of using directly?

Reply:
To make the code less verbose, obviously you can use it directly if this is the convention you usually use maybe it's better not to change it, how about that? Do we use it directly or leave it as is?

Reply:
Up to you, thought I'd mention

Reply:
I would leave it like that, with tailwind with the use of "after" we would have a very verbose css class